README.md

```diff
@@ -203,6 +205,15 @@ pip3 install -r tests/requirements.txt
 python tests/run_tests.py
 ```
 
+### Fuzzing
+
+This project uses `cargo-fuzz` (libfuzzer) for fuzzing, you can run a fuzz target with:
+```bash
+cargo +nightly fuzz run local_address_str
+```
+
+You can replace `local_address_str` with the name of any other target you want to run.
+
```

Comment:
Wouldnt be great to have links to more info, like addressing the link to the project
(https://github.com/rust-fuzz/cargo-fuzz).
or more information about fuzzing in general ? 

I think that this type of bloating is never bad :D

Reply:
Sounds good, I can do it in a follow-up PR.